src/hooks/useClickableProps/__stories__/useClickableProps.stories.mdx

```diff
@@ -0,0 +1,275 @@
+import { useCallback, useRef } from "react";
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Link, Tip } from "../../../storybook";
+import useClickableProps from "../useClickableProps";
+import "./useClickableProps.stories.scss";
+
+<Meta title="Hooks/useClickableProps" />
+
+# useClickableProps
+
+- [Overview](#overview)
+- [Arguments](#arguments)
+- [Returns](#returns)
+- [Feedback](#feedback)
+
+## Overview
+
+Return props for adding clickable functionality to the element except for the styles and classNames.
```

Comment:
```suggestion
Return props for making an element or component clickable by mouse and keyboard with screen reader support.
```

The important thing about this hook is that it make element to clickable in accessible way. It's less important to mention that we are not supporting style props because it's not related to the hook's purpose